app/qml/form/FeatureFormPage.qml

```diff
@@ -236,17 +235,19 @@ Item {
         }
       }
 
-      AutoHideBanner {
-        id: editingFailedBanner
-
-        width: featureForm.width - InputStyle.innerFieldMargin * 2
-        height: InputStyle.rowHeight
-
-        bgColor: InputStyle.warningBannerColor
+      MessageDialog {
+        id: editingFailedDialog
 
-        source: InputStyle.noIcon
+        visible: false
+        title: qsTr( "Saving failed" )
+        text: qsTr( "Failed to save changes. This should not happen normally. Please restart the app and try again — if that does not help, please contact support." )
+        icon: StandardIcon.Warning
+        standardButtons: StandardButton.Close
 
-        text: qsTr( "Failed to save changes. Try to restart app." )
+        //! Using onButtonClicked instead of onAccepted,onRejected which have been called twice
+        onButtonClicked: {
+          visible = false
```

Comment:
Let's keep it like this, but generally it is better to use `.open()` and `.close()` (instead of `visible = true/false`) because it does not break property binding.

Reply:
I have followed the approach used in the existing dialogs. Do you want me to update all occurrences in this file?

Reply:
Let's keep it like this :)